server/src/main/java/net/hypejet/jet/server/plugin/JetPluginManager.java

```diff
@@ -217,6 +240,34 @@ private void load(@NonNull PluginPair pluginPair, @NonNull Map<String, PluginPai
         }
     }
 
+    private static boolean isCompatible(@NonNull String versionString, @NonNull PluginDependency dependency) {
+        for (PluginVersion version : dependency.versionsSupported())
+            if (isVersionCompatible(versionString, version))
+                return true;
+        return false;
+    }
+
+    private static boolean isVersionCompatible(@NonNull String versionString, @NonNull PluginVersion requiredVersion) {
```

Comment:
`*`  in required version should bypass this function and return true

fx this in the top of the function
``if (requiredVersion.equals("*")) return true`` (of course with whitespace stripping)